### PR TITLE
Exposure: tests and download documents

### PR DESCRIPTION
**Pull request by hascar**
Wrote the tests, changed util and view py for to pass from fake ged and fixed problems with download csv fractions
The tests are green here: https://ci.openquake.org/job/zdevel_oq-platform2/149/

### CODE REVIEW

File: openquakeplatform/exposure/views.py [hunk 1]
```diff
@@ -199,33 +202,39 @@ def export_exposure(request):
...
-        return response
+        
     output_type = request.GET.get('output_type')
     if not output_type or output_type not in ('csv', 'nrml'):
         msg = ('Please provide the parameter "output_type".'
                ' Available choices are "csv" and "nrml".')
         response = HttpResponse(msg, status="400")
         return response
+
+    content_disp_type = ('attachment' if 'geddb' in connections else 'inline')
```
Comment (nastasi-oq) [inline openquakeplatform/exposure/views.py]: With the new `moon` probably we can return to use `attachment` as in master.

File: openquakeplatform/static/exposure/exposure_controller.js [hunk 2]
```diff
@@ -54,18 +54,7 @@ var showErrorDialog = function(message, options) {
...
-        success: function(data, textStatus, jqXHR) {
-            if (navigator.appName != 'Microsoft Internet Explorer') {
-                window.open('data:text/csv;charset=utf-8,' + escape(data));
-            } else {
-                var popup = window.open('','csv','');
-                popup.document.body.innerHTML = '<pre>' + data + '</pre>';
-            }
-        }
-    });
+    window.open('export_fractions_by_study_region_id?sr_id='+srId);
```
Comment (nastasi-oq) [inline openquakeplatform/static/exposure/exposure_controller.js]: Add a comment about the change and what it implies about empty new tabs (and the fact that the right solution is using the StreamingResponse)

File: openquakeplatform/static/geonode/css/oqplatform_leaflet.css [hunk 3]
```diff
@@ -1,5 +1,10 @@
+#wrap,
+#wrap .container {
+    width: 100%;
+}
 #oq-page-middle {
-    background: #555555;
+    background: #fff;
```
Comment (nastasi-oq) [inline openquakeplatform/static/geonode/css/oqplatform_leaflet.css]: Could we continue to use 6 chars syntax for `#rrggbb` colors ?

File: openquakeplatform/templates/exposure/export.html [hunk 4]
```diff
@@ -46,7 +46,7 @@
     <div id="national-spinner" >Loading ...<img id="download-button-spinner" src="{{ STATIC_URL }}img/ajax-loader.gif" /></div>
     {% verbatim %}
       <table ng-table="tableParams" show-filter="true" class="table ng-table-rowselected">
-        <tr ng-repeat="record in $data" ng-click="record.$selected = !record.$selected; changeSelection(record)"
+        <tr class="{{record.country_name}}{{record.num_studies}}" ng-repeat="record in $data" ng-click="record.$selected = !record.$selected; changeSelection(record)"
```
Comment (nastasi-oq) [inline openquakeplatform/templates/exposure/export.html]: Could we use a different attribute to match `tr` ?

File: openquakeplatform/exposure/views.py (hunk 1 above)
Comment (hascar) [reply openquakeplatform/exposure/views.py]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/aebefdb84cc50dedcf77bcf1473c17f03c3cd0fb

File: openquakeplatform/static/geonode/css/oqplatform_leaflet.css (hunk 3 above)
Comment (hascar) [reply openquakeplatform/static/geonode/css/oqplatform_leaflet.css]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/63fbc4a24a06952253b7cad96c4898adb28970c4

File: openquakeplatform/static/exposure/exposure_controller.js (hunk 2 above)
Comment (hascar) [reply openquakeplatform/static/exposure/exposure_controller.js]: Fixed here: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/a590f2ef001b310fd7446306c1ff997a0d186113

Comment (hascar) [reply openquakeplatform/static/exposure/exposure_controller.js]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/b138645df415e30139dd5747d6b2a742f1876d97

File: openquakeplatform/static/geonode/css/oqplatform_leaflet.css (hunk 3 above)
Comment (hascar) [reply openquakeplatform/static/geonode/css/oqplatform_leaflet.css]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/a5bb6bdd11c8f111a993446fc3b83f86ee5fe9b4

File: openquakeplatform/exposure/views.py (hunk 1 above)
Comment (hascar) [reply openquakeplatform/exposure/views.py]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/218f9c64361907fb13bb83ea8b11c6901cca11b2

File: openquakeplatform/templates/exposure/export.html (hunk 4 above)
Comment (hascar) [reply openquakeplatform/templates/exposure/export.html]: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/aab0358c7d2def982eebfb9b57a8158e483544d9

**Comment by nastasi-oq** [comment]
Test are green here: https://ci.openquake.org/job/zdevel_oq-platform2/149/

**Pull request merged**